packages/react-components/src/organisms/ManuscriptCard.tsx

```diff
@@ -148,6 +161,27 @@ const ManuscriptCard: React.FC<ManuscriptCardProps> = ({ title, versions }) => {
                 <Pill accent="gray">{version.type}</Pill>
                 <Pill accent="gray">{version.lifecycle}</Pill>
               </div>
+              <div>
+                <Divider />
+
+                <div css={fileContainerStyles}>
+                  {linkIcon}
+                  <Subtitle>{version.manuscriptFile.filename}</Subtitle>
+                  <div css={css(downloadButtonStyles)}>
+                    <Link
+                      href={version.manuscriptFile.url}
+                      primary
+                      noMargin
+                      small
+                      buttonStyle
+                    >
+                      {downloadIcon} Download
+                    </Link>
+                  </div>
+                </div>
+
+                <Divider />
```

Comment:
I believe we need to check if there is at least on quick check that will render so we don't have these dividers duplicated as shown in the right side
<img width="1370" alt="Screenshot 2024-07-15 at 09 45 51" src="https://github.com/user-attachments/assets/6fc08c73-0ac6-49e3-a7bd-4ca451fd5613">